MinStack.java

```diff
@@ -0,0 +1,39 @@
+// Time Complexity : O(1)
+// Space Complexity : O(1)
+// Did this code successfully run on Leetcode : Yes
+// Any problem you faced while coding this : No
+
+class MinStack {
```

Comment:
Please error guard pop(), top() & getMin() methods against operating on an empty stack, the program will crash otherwise in certain test cases.